_overviews/scala3-book/scala-tools.md

```diff
@@ -92,31 +119,33 @@ A nice benefit of that is that once you’re comfortable with its structure, it
 
 The first thing to know is that underneath the root directory of your project, sbt expects a directory structure that looks like this:
 
-```bash
-build.sbt
-project/
-src/
--- main/
-   |-- java/
-   |-- resources/
-   |-- scala/
-|-- test/
-   |-- java/
-   |-- resources/
-   |-- scala/
-target/
+```text
+.
+├── build.sbt
+├── project/
```

Comment:
Should we add _build.properties_ under the _project_ directory here as well?

Reply:
Added.